.github/workflows/container-image.yml

```diff
@@ -177,3 +188,43 @@ jobs:
           subject-name: index.docker.io/${{ env.IMAGE_NAME }}
           subject-digest: ${{ steps.build-and-push.outputs.digest }}
           push-to-registry: false
+
+      - name: Prepare For-Container.md file
+        if: ${{ env.LATEST == 'true' }}
+        run: |
+          if [ -z "${{ env.README_FILEPATH }}" ]; then
+            file_path=$(find ./doc/ -type f -name 'For-Container.md' | head -n 1)
+            if [ -z "$file_path" ]; then
+              echo "No For-Container.md file found in the ./doc/ directory."
+              exit 1
+            fi
```

Comment:
I'd change the implementation so that either `README_FILEPATH` is given, or there must be exactly one `For-Container.md` inside `doc/`.

Hint: if you do the following, you'll have all matches in an array and can easily check its length:

```bash
shopt -s globstar # enable ** for recursive glob search
files=a=(doc/**/For-Container.md)
```

Reply:
```bash
$ shopt -s globstar
$ files=a=$(doc/**/For-Container.md)
bash: doc/02-installation.md.d/For-Container.md: Permission denied
```

Reply:
Oh, of course! Ignore my previous comment :)! This is the actual error I get when I copy paste your commands:
```bash
$ shopt -s globstar
$ files=a=(doc/**/For-Container.md)
bash: syntax error near unexpected token `('
```

Reply:
I'm too stupid to copy & paste :sweat_smile: I tried this locally with variable name `a` then thought "would be nice to use a proper variable name" and prepended it instead of replacing it. There should be no `a=` in there, see my edited comment above now.